doc/connection.rst

```diff
@@ -122,12 +124,30 @@ Connection Types
     :param connection_attempts: Specify how many connection attempts, 10 seconds
                                 apart, should be attempted to connect to the device.
                                 Defaults to 5.
+    :param poll_transfers: Specify whether file transfers should be polled. Polling
+                           monitors the progress of file transfers and periodically
+                           checks whether they have stalled, attempting to cancel
+                           the transfers prematurely if so.
+    :param start_transfer_poll_delay: If transfers are polled, specify the length of
+                                      time after a transfer has started before polling
+                                      should start.
+    :param total_transfer_timeout: If transfers are polled, specify the total amount of time
+                                   to elapse before the transfer is cancelled, regardless
+                                   of its activity.
+    :param transfer_poll_period: If transfers are polled, specify the period at which
+                                 the transfers are sampled for activty.
+    :param activity_sample_interval: If transfers are polled, specify the interval
+                                     between the first and the second sample of the 
+                                     destination path's size.
```

Comment:
Might it be worth adding a warning to the relevant parameters that setting a too low value can cause it to timeout unnecessarily?